source/reference/mongosync.txt

```diff
@@ -176,6 +176,38 @@ Global Options
 
    Prints ``mongosync`` version information to stdout.
 
+.. _c2c-cli-oidc:
+
+Microsoft Azure Managed Identities Environment Variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/mongosync-and-oidc.rst
+
+To use Microsoft Azure Managed Identities for federated authentication,
+define these environment variables:
+
+.. option:: AZURE_TENANT_ID <id>
+
+   Sets the Azure tenant identifier. Example:
+   ``AZURE_TENANT_ID=08206ab8-16a0-406d-85e4-2f15f5620fac``
+
+.. option:: AZURE_APP_CLIENT_ID <id>
+
+   Sets the Azure application client identifier. Example:
```

Comment:
Likewise: were I a user, I would expect some guidance on where to find the value to plug in here.